docs/README.md

```diff
@@ -215,3 +215,14 @@ contribution](https://trino.io/development/process.html).
    Docker](https://www.youtube.com/watch?v=y58sb9bW2mA) gives you a starting
    point for setting up a test system on your laptop.
 
+## Release process
+
+Trino aims to release once per week on Wednesday. This is a goal but not a
```

Comment:
I don't think we want to commit to anything about Wednesdays.

in the general README it should be fine to say that "Trino aims for frequent releases".